tests/test_trigger_bisect_jobs.py

```diff
@@ -197,6 +200,13 @@ def test_problems():
     openqa.main(args)
     openqa.openqa_clone.assert_not_called()
 
+    args.url = "http://openqa.opensuse.org/tests/101"
+    try:
+        openqa.main(args)
+    except FileNotFoundError as e:
```

Comment:
I don't think we need to test an exception coming from a mocked function.
It should never occur, and if it does, then someone forgot to add the file to the test data.
So I don't see what's the motivation for this test.

Reply:
The motivation is checking if the script returns if the job result is "passed".